### PR TITLE
Support bot token

### PR DESCRIPTION
**Pull request by whywaita**
Add support using [bot token](https://api.slack.com/authentication/token-types#bot) for posting messages.

### CODE REVIEW

File: handler.go [hunk 1]
```diff
@@ -31,8 +33,8 @@ func (o Option) NewSlackHandler() slog.Handler {
 		o.Level = slog.LevelDebug
 	}
 
-	if o.WebhookURL == "" {
-		panic("missing Slack webhook url")
+	if o.WebhookURL == "" && o.BotToken == "" {
+		panic("missing Slack webhook url and bot token")
```
Comment (samber) [inline handler.go:37]: and -> or

File: README.md [hunk 2]
```diff
@@ -114,6 +116,27 @@ func main() {
 }
 ```
 
+#### Bot token
```
Comment (samber) [inline README.md:119]: On top of existing example, can you add a title please?

```
#### Using webhook

Generate a webhook [here](https://slack.com/apps/A0F7XDUAZ-incoming-webhooks).
```

```
#### Using bot token
```

**Review by samber — CHANGES_REQUESTED** [review]
Hi @whywaita and thanks for your contrib!

2 little change and LGTM ;)

File: handler.go (hunk 1 above)
Comment (whywaita) [reply handler.go:37]: It needs one of the URLs of webhook or bot token.
So need panic if it is blank.

**Comment by samber** [comment]
Available in version 0.3.0 👌

**Comment by whywaita** [comment]
Thank you!

**Pull request merged**